tests/test_apel_plugin.py

```diff
@@ -108,3 +110,43 @@ async def test_get_time_db(self):
             os.remove(path)
 
             assert result == [(time_stamp, datetime(1970, 1, 1, 0, 0, 0))]
+
+    async def test_sign_msg(self):
+        result = await sign_msg(
```

Comment:
What is `result`?

Reply:
`result` is the signed message

Reply:
This question of mine is the result of the confusion I had above. Can be ignored.